Source/WebCore/platform/graphics/ColorTypes.h

```diff
@@ -201,6 +201,16 @@ constexpr bool operator==(const ColorType& a, const ColorType& b)
 
 // MARK: - RGB Color Types.
 
+enum class ColorSpace : uint8_t;
+
+struct OutOfLineColorData {
+    ColorSpace colorSpace;
+    float c1;
```

Comment:
Please spell out red, green, blue instead of c1, c2, c3.

Reply:
This is not rgb data, it is just generic 3 channel color, so c1, c2, c3 is the canonical naming.